feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/TimetableScreen.kt

```diff
@@ -207,6 +222,11 @@ private fun TimetableScreen(
                 uiState = uiState.contentUiState,
                 timetableScreenScrollState = state,
                 onFavoriteClick = onBookmarkClick,
+                windowInsetsPadding = PaddingValues(
+                    bottom = innerPadding.calculateBottomPadding(),
```

Comment:
May I ask if `innerPadding.calculateBottomPadding()` contains OS's navigatiaon bar and bottom navigation size? 🙋

Reply:
I don't understand yet 🙇

Reply:
What is passed from Scaffold is a recognition that the sizes of Systembar and BottomBar are combined.

Reply:
This naming is out of sync with the others and should be changed.
windowInsetsPadding → contentPadding

Reply:
So the inner layout has the bigger padding, right?

Would it be good to adapt using rules like these?

## The Policy
Insets are not consumed; instead, they are added. As a result, the inner padding will be larger.

## The Rule
Insets should be passed as parameters to screen-level Composable functions.
Composable functions for screen containers, such as MainScreen, should add insets to child screens and pass them along.

Reply:
I think it's a good rule to implement EdgeToEdge, Tablet and Mobile layouts.

Reply:
Great!